ChangeLog

```diff
@@ -74,6 +74,7 @@
 	  Thanks to Pavel Rochnyak. #2570, #2673
 	* Write Syslog plugin: The new "write_syslog" plugin writes value
 	  lists as syslog messages. Thanks to Shirly Radco. #3019
+	* Write Ubi plugin: The new "read_ubi" plugin reads value for UBIFS.
```

Comment:
Hi,

Your plugin is not part of `Collectd-5.8.1`, so this line is wrong.
Moreover, please don't touch Changelog, that file is updated semi-automatically while new release preparation, based on "Changelog" line in PR description.

Reply:
The next:

`Write Ubi plugin: The new "read_ubi" plugin reads value for UBIFS.`

"Write Ubi" was written based on "Write Syslog" or similar. 
If you read "Write Syslog" line carefully, you find that plugin writes values to syslog.
Your plugin does not write metrics, it _reads_ them.

`The new "read_ubi" plugin reads value for UBIFS.`

1) Your plugin is not "read_ubi".
2) "reads value for UBIFS" is not informative.

Probably, Changelog line should be something like "The new "ubifs" plugin reports block state for flash memory devices with UBIFS filesystem".   As you more familar with this, you can write this shorter and  more precise.

Reply:
Ok, thank you for the clarification.